docs/upstream-integrations/new-bot-command/aws-vault-setup.md

```diff
@@ -120,8 +120,3 @@ rm ~/.aws/credentials
 You can change password or settings for the `aws-vault` key chain. See
 instructions at
 https://github.com/99designs/aws-vault/blob/master/USAGE.md#keychain.
-
```

Comment:
Did you resolve this error before deleting reference?

Reply:
Yes